### PR TITLE
PRT: Adding a success debug print to CUC

### PR DESCRIPTION
**Pull request by oneeuroquestion**

### CODE REVIEW

**Review by Yaroms — APPROVED** [review]

File: protocol/metrics/consumer_relayserver_client.go [hunk 1]
```diff
@@ -111,19 +113,21 @@ func (cuc *ConsumerRelayServerClient) SetRelayMetrics(relayMetric *RelayMetrics)
 	cuc.appendQueue(request)
 }
 
-func (cuc *ConsumerRelayServerClient) aggregateAndSendRelayData(sendQueue []UpdateMetricsRequest, sendID int, cucEndpointAddress string) (*http.Response, error) {
+func (cuc *ConsumerRelayServerClient) aggregateAndSendRelayData(sendQueue []UpdateMetricsRequest, sendCnt int, cucEndpointAddress string) (*http.Response, error) {
+	utils.LavaFormatDebug("[CUC] Sending data to server - start ", utils.LogAttr("Size of metrics to send", len(sendQueue)))
```
Comment (ranlavanet) [inline protocol/metrics/consumer_relayserver_client.go:117]: please dont print every time you aggregate, just when you send

**Review by ranlavanet — CHANGES_REQUESTED** [review]

File: protocol/metrics/consumer_relayserver_client.go [hunk 2]
```diff
@@ -132,48 +136,49 @@ func (cuc *ConsumerRelayServerClient) aggregateAndSendRelayData(sendQueue []Upda
...
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
 		bodyBytes, err := io.ReadAll(resp.Body)
 		if err != nil {
-			utils.LavaFormatWarning("[CUC] failed reading response body", err)
+			utils.LavaFormatWarning("[CUC] Sending data to server - failed reading response body", err)
 		} else {
-			utils.LavaFormatWarning("[CUC] Received non-200 status code", nil, utils.LogAttr("status_code", resp.StatusCode), utils.LogAttr("body", string(bodyBytes)))
+			utils.LavaFormatWarning("[CUC] Sending data to server - Received non-200 status code", nil, utils.LogAttr("status_code", resp.StatusCode), utils.LogAttr("body", string(bodyBytes)))
```
[the remark this answers is not in the document]
Comment (oneeuroquestion) [reply protocol/metrics/consumer_relayserver_client.go:167]: should this be a debug print ?

File: protocol/metrics/consumer_relayserver_client.go (hunk 1 above)
Comment (oneeuroquestion) [reply protocol/metrics/consumer_relayserver_client.go:117]: i was debugging it with nenad last time and starting to send was also valuable infromation - then you see if the problem that the sending takes 2 much time
what are my options to turn them on/off for debugging ?

**Comment by ranlavanet** [comment]
@oneeuroquestion lets add a flag to the rpcconsumer process called --analytics-server-logs

**Comment by omerlavanet** [comment]
how about instead we will use QueueSender class in the relayServer client?

**Comment by ranlavanet** [comment]
@oneeuroquestion please see omer's comment when you have the time.

**Comment by ranlavanet** [comment]
closed for being unprioritized at the moment.

**Pull request closed without merging**